drivers/mlx90393/include/mlx90393_params.h

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (C) 2023 mi6527ri
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_mlx90393
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for MLX90393 device driver
+ *
+ * @author      Michael Ristau <michael.ristau@fh-erfurt.de>
+ */
+
+#ifndef MLX90393_PARAMS_H
+#define MLX90393_PARAMS_H
+
+#include "board.h"
+#include "mlx90393.h"
+#include "mlx90393_constants.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Set default configuration parameters for the MLX90393
+ * @{
+ */
+#ifndef MLX90393_PARAM_MODE
+/**
+ * @brief   Default measurement mode
+ */
+#define MLX90393_PARAM_MODE             (MLX90393_MODE_BURST)
+#endif
+
+#if IS_USED(MODULE_MLX90393_INT) || DOXYGEN
+#ifndef MLX90393_PARAM_INT_PIN
+/**
+ * @brief   Default interrupt pin
+ */
+#define MLX90393_PARAM_INT_PIN          .int_pin = (GPIO_PIN(PORT_C, 8)),
+#endif
+#else
+#define MLX90393_PARAM_INT_PIN
+#endif
```

Comment:
So kann man das leider nicht realisieren, kein Mensch wird an der `make`-Kommandozeile ein solches Konstrukt angeben können/wollen, um den Default-Wert zu überschreiben.

```suggestion
/**
 * @brief   Default interrupt pin
 */
#ifndef MLX90393_PARAM_INT_PIN
#define MLX90393_PARAM_INT_PIN          (GPIO_PIN(PORT_C, 8))
#endif

#if IS_USED(MODULE_MLX90393_INT) || DOXYGEN
#define MLX90393_PARAM_INT             .int_pin = MLX90393_PARAM_INT_PIN,
#else
#define MLX90393_PARAM_INT
#endif
```

In der Definition von `MLX90393_PARAM_SPI` bzw. `MLX90393_PARAM_I2C` wird dann einfach grundsätzlich `MLX90393_PARAM_INT` hinzugefügt, was entweder leer ist oder ebent die Deklaration des `int_pin` Members enthält, siehe unten.